component/prometheus/exporter/cloudwatch/config.go

```diff
@@ -240,3 +243,13 @@ func toYACEDiscoveryJob(rj DiscoveryJob) *yaceConf.Job {
 	}
 	return job
 }
+
+// getHash calculates the MD5 hash of the river representation of the config.
+func getHash(a Arguments) string {
```

Comment:
I understand this is copying static mode, but this seems utterly ridiculous to me. I wish there was some more usable version of this. Would the component ID be preferable in this case?

Reply:
Maybe yes, I don't have a preference, but I'd like to set it to something more representative of what's being monitored instead of just the component definition.